redshift/redshift.go

```diff
@@ -19,59 +17,55 @@ type dbExecCloser interface {
 	Exec(query string, args ...interface{}) (sql.Result, error)
 }
 
+// S3Info holds the information necessary to copy data from s3 buckets
+type S3Info struct {
+	Region    string
```

Comment:
good point - this will actually be pretty easy to fix when I do the final 3/3 PR here, but good feedback!